sql/catalyst/src/main/scala/org/apache/spark/sql/Encoders.scala

```diff
@@ -109,13 +109,27 @@ object Encoders {
    */
   def DATE: Encoder[java.sql.Date] = ExpressionEncoder()
 
+  /**
+   * An encoder for nullable date type.
```

Comment:
Hi, @MaxGekk . Do you think we can have a more distinguishable description here? For now, this is the same with line 106.

Reply:
I have changed the comments here and below. Thanx.